README.md

```diff
@@ -1,32 +1,144 @@
 # DataRobot User Models
 
-## What is it?
-**The DataRobot User Models** repository contains boilerplate, information, and tools on how to assemble,
-debug, test, and run your training and inference models on the DataRobot platform.
-
-### Terminology
-Described DataRobot functionality is known as `Custom Model`, so words `custom model` and `user model` may be interchangeably used.
-Also `custom model directory` and `code directory` mean the same entity.
-
 
 ## Content  
-- [model templates](model_templates) - contains templates for building and deploying custom models in DataRobot.
-- [environment templates](public_dropin_environments) - contains templates of base environments used in DataRobot. Dependency requirements can be applied to the base to create runtime environment for custom models.
-- [user models runner](custom_model_runner) - **drum** - a tool that helps to assemble, test, and run custom model. For more information about how to run drum, check out the [pypi docs](https://pypi.org/project/datarobot-drum/)
-
-## Custom Model Templates
-This repository contains templates for building and deploying custom models in DataRobot.
-
-Custom Inference Models are models that are trained outside of DataRobot. Once they're uploaded to DR, they are deployed straight to a DR Deployment, and tracked with Model Monitoring and Management.
-
-Custom Training Models are in active development. They include a `fit()` function, and can be trained on the Leaderboard, benchmarked against DR AutoML models, and get access to our full set of automated insights. Check out [The quickrun readme here](QUICKSTART-FOR-TRAINING.md)
+1. [What is this repository?](#what_is_it)
+2. [Quickstart and examples](#quickstart)
+3. [Assembling inference model code folder](#inference_model_folder)
+4. [Custom Model Templates](#custom_model_templates)
+5. [Custom Environment Templates](#custom_environment_templates)
+6. [Custom Model Runner (drum)](#custom_model_runner)
+4. [Contribution & development](#contribution_development)
+
+## What is this repository? <a name="what_is_it"></a>
+The **DataRobot User Models** repository contains information and tools for assembling,
+debugging, testing, and running your training and inference models with DataRobot.
+
+#### Terminology
+This repository address the DataRobot functionality known as `custom models`. The terms `custom model` and `user model` can be used interchangeably, as can `custom model directory` and `code directory`.
```

Comment:
I thought the terms were different... a custom model directory is a code directory with a model in it.

Reply:
is custom model directory still used?

Reply:
yes, it is used.
If we want to unify, let's make it `code dir` everywhere: in training and in inference.
But people will continue to calling it `custom model dir`, so better to have this disclaimer.